outlines/generate/regex.py

```diff
@@ -3,6 +3,7 @@
 from outlines.fsm.fsm import RegexFSM
 from outlines.generate.api import SequenceGenerator
 from outlines.models import OpenAI
+from outlines.models.llamacpp import LlamaCpp, RegexLogitsProcessor
```

Comment:
We should import within `def regex_llamacpp`, otherwise using `outlines.generate.regex` and `outlines.generate.json` will require installing `llama_cpp_python`

Reply:
We register on the `LlamaCpp` type which is a subclass of `Llama`, so there's clearly a design issue here